app/resources/views/pages/profile/profile.blade.php

```diff
@@ -3,11 +3,19 @@
   <link href="/css/jquery/jquery-ui.css" rel="stylesheet" type="text/css"> 
   <script src="/css/jquery/external/jquery/jquery.js"></script>
   <script src="/css/jquery/jquery-ui.js"></script>
-  <script src="/js/profile.js"></script>
+  <script src ="/js/profile_photo_upload.js"> </script>
+    <script src ="/css/jquery/"></script>
+   <!--<script src="/js/profile.js"></script>--> 
+   
 <link href="/css/app.css" rel="stylesheet" type="text/css">
 @stop
 @section('content')
-
+<script type="text/javascript">
+   function funct(m)
```

Comment:
"funct" isn't a very clear description for what that function does.  Maybe "setDbText" is better.

Rename the function to be more descriptive.

Also, move this function to profile_photo_upload.js if it is used there.